ruby_review_week_four.rb

```diff
@@ -1,22 +1,55 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
+# a variable is an object with a value
```

Comment:
a variable is technically an object but this definition isn't perfect, we could say something like 

> a variable is used to store value, and that value is then reusable in our code by referencing the name we gave to the variable